phoenicis-apps/src/main/java/org/phoenicis/apps/RepositoryManager.java

```diff
@@ -57,8 +59,12 @@ public void setOnError(Consumer<Exception> onError) {
         this.onError = onError;
     }
 
-    public Repository cachedRepository() {
-        return this.cachedRepository;
```

Comment:
The implementation of the RepositoryManager is adjusted to the underlying repository architecture. This architecture looks currently like this:

> MultipleRepository -> FilterRepository -> CachedRepository -> BackgroundRepository

The internal variables for the 4 different repositories (which are each saved in their own variable) are `multipleRepository`, `filterRepository`, `cachedRepository` and `backgroundRepository`.

From outside we currently have possibility to indirectly access the FilterRepository, CachedRepository and BackgroundRepository through specialized methods.
The important part is, that the access is only indirectly possible, because the variables are marked as private and there exist no getter and setter methods.

Therefore I would suggest, if we change the repository structure inside the RepositoryManager at a later time, we change the variable names too.